packages/data-provider/src/services/dictionaryService.ts

```diff
@@ -81,6 +88,15 @@ const dictionaryService = (dependencies: BaseDependencies) => {
 
 		const dictionary = await fetchDictionaryByVersion(dictionaryName, version);
 
+		if (
+			defaultCentricEntity &&
+			defaultCentricEntity.length > 0 &&
```

Comment:
think this probably covers both checks
```suggestion
			defaultCentricEntity?.length > 0 &&
```

Reply:
This comment will be handled in PR[#105](https://github.com/overture-stack/lyric/pull/105)
Code changed as suggested! thanks!